src/tun2proxy.rs

```diff
@@ -244,6 +244,9 @@ pub(crate) trait ConnectionManager {
 
 const TCP_TOKEN: Token = Token(0);
 const UDP_TOKEN: Token = Token(1);
+const EXIT_TOKEN: Token = Token(34255);
```

Comment:
This should maybe be `Token(2)`. Then we could start with 3 here:

https://github.com/blechschmidt/tun2proxy/blob/7442abece583b19ac21e8e858a70c665d25d0403/src/tun2proxy.rs#L309

Otherwise, we will have a future collision here:
https://github.com/blechschmidt/tun2proxy/blob/7442abece583b19ac21e8e858a70c665d25d0403/src/tun2proxy.rs#L512-L513